internal/controller/instaslice_daemonset.go

```diff
@@ -180,25 +168,26 @@ func (r *InstaSliceDaemonsetReconciler) Reconcile(ctx context.Context, req ctrl.
 			//Assume pod only has one container with one GPU request
 			log.FromContext(ctx).Info("creating allocation for ", "pod", allocations.PodName)
 			var podUUID = allocations.PodUUID
-			deviceForMig, profileName, resourceIdentifier, errGettingControllerAllocation := r.getAllocation(instaslice, allocations.PodUUID)
+			_, profileName, resourceIdentifier, errGettingControllerAllocation := r.getAllocation(instaslice, allocations.PodUUID)
 			if errGettingControllerAllocation != nil {
-				log.FromContext(ctx).Error(errGettingControllerAllocation, "allocation was not found, retrying will not help")
+				//allocation was not found, retrying will not help
```

Comment:
IMHO, it won't hurt to log here with the error `errGettingControllerAllocation`

Reply:
Thanks, Done.